dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/ControlledVocabularyValueConverter.java

```
@@ -35,7 +35,11 @@ public String getAsString(FacesContext facesContext, UIComponent component, Obje
         if (value == null || value.equals("")) {
             return "";
         } else {
```

This else is not needed - just the first if() is needed -> fail fast approach.